src/index.ts

```diff
@@ -1,9 +1,10 @@
 import "reflect-metadata";
+const uuid = require("uuid/v4");
```

Comment:
this is nitpicky but if were going with imports, id rather keep them consistent.
`import uuid from "uuid/v4";`

Reply:
cool I'll change this